Formula/libimobiledevice.rb

```diff
@@ -20,20 +20,76 @@ class Libimobiledevice < Formula
     depends_on "libxml2"
   end
 
+  option "with-python", "Enable Cython Python bindings"
+  option "with-python3", "Enable Cython Python3 bindings"
+
   depends_on "pkg-config" => :build
   depends_on "libtasn1"
-  depends_on "libplist"
   depends_on "usbmuxd"
   depends_on "openssl"
+  depends_on "libplist" => (build.with?("python") ? "with-python" : []) || (build.with?("python3") ? "with-python3" : [])
+  depends_on :python => :optional
+  depends_on :python3 => :optional
+
+  resource "cython" do
+    url "http://cython.org/release/Cython-0.24.tar.gz"
+    sha256 "6de44d8c482128efc12334641347a9c3e5098d807dd3c69e867fa8f84ec2a3f1"
+  end
+
+  patch :DATA
 
   def install
+    ENV.deparallelize
```

Comment:
This should also be reported upstream as a bug. Things _should_ build in parallel.